src/content/api/sales-channels.mdoc

```diff
@@ -0,0 +1,140 @@
+    The id of the Sales Channel for the device provider. Required if `type` is `reseller`.
+  {% /property %}
+
+  {% property name="deviceId" type="string" %}
+    `true` if the device id is required to onboard to Sales Channel.
+  {% /property %}
+
+  {% property name="deviceModel" type="string" %}
+    `true` if the device model is required to onboard to Sales Channel.
+  {% /property %}
+
+  {% property name="externalId" type="string" %}
+    `true` if an external id is required to onboard to Sales Channel.
+  {% /property %}
+
+  {% property name="notificationEmail" type="string" %}
+    Email to notify for requests to connect to Sales Channel.
+  {% /property %}
+
+  {% property name="frequency" type="string" %}
+    Frequency of notification emails valid values are `weekly` and `daily`. Requred if `notificationEmail` is set.
+  {% /property %}
+
+  {% property name="notificationUrl" type="string" %}
+    Url for webhook notifications for requests to connect to Sales Channel.
+  {% /property %}
+
+  {% property name="createdAt" type="timestamp" %}
+    When the Sales Channel was created.
+  {% /property %}
+
+  {% property name="createdBy" type="crn" %}
+    The User or API Key that created the Sales Channel.
+  {% /property %}
+
+  {% property name="updatedAt" type="timestamp" %}
+    When the Sales Channel was updated.
+  {% /property %}
+
+  {% property name="updatedBy" type="crn" %}
+    The User or API Key that updated the Sales Channel.
+  {% /property %}
+{% /properties %}
+
+---
+
+{% endpoint
+  path="/api/sales-channels"
+  filename="sales-channels-create"
+%}
+  ## Create Sales Channel {% badge type="experimental" /%}
+
+  This endpoint allows you to create an [Sales Channel](#sales-channel-model).
+
+  {% properties %}
+    {% property name="name" type="string" required=true %}
+      The display name of the Sales Channel.
+    {% /property %}
+
+    {% property name="type" type="string" required=true %}
+      The type of the Sales Channel integration either `reseller` or `provider`.
+    {% /property %}
+
+    {% property name="accountId" type="string" required=true %}
+      Id of the [Account](/api/accounts/) that owns the Sales Channel.
+    {% /property %}
+
+    {% property name="test" type="boolean" %}
+      `true` if the Asset Program is for testing purposes only. This is based on the Account that owns the Asset Program.
+    {% /property %}
+
+    {% property name="providerSalesChannelId" type="string" %}
+      The id of the Sales Channel for the device provider. Required if `type` is `reseller`.
+    {% /property %}
+
+    {% property name="deviceId" type="string" %}
+      `true` if the device id is required to onboard to Sales Channel.
+    {% /property %}
+
+    {% property name="deviceModel" type="string" %}
+      `true` if the device model is required to onboard to Sales Channel.
+    {% /property %}
+
+    {% property name="externalId" type="string" %}
```

Comment:
This is not on the endpoint - should it be?

Reply:
Maybe I will remove for now